Sources/KituraSession/SessionState.swift

```diff
@@ -76,6 +76,8 @@ public class SessionState {
             store.save(sessionId: id, data: data, callback: callback)
         } catch(let error as NSError) {
             callback(error)
+        } catch {
+            callback(nil)
```

Comment:
If nothing else, I think it should include file name/line number for debugging.

I'm surprised that it wasn't an issue in 3.0. Aren't catch blocks supposed to be exhaustive, like switch blocks, if the enclosing function isn't throwing? My understanding is that the Swift compiler somehow knows that the all errors that can be thrown in that try expression are castable to NSError (hence that one catch block is exhaustive), and this changed for Linux on 3.1.